napari/_qt/_qapp_model/_tests/test_view_menu.py

```diff
@@ -189,3 +189,27 @@ def test_toggle_layer_tooltips(make_napari_viewer, qtbot):
     # Restore layer tooltip visibility
     app.commands.execute_command(action_id)
     assert not _get_current_tooltip_visibility()
+
+
+def test_zoom_actions(make_napari_viewer):
+    """Test zoom actions"""
+    viewer = make_napari_viewer()
+    app = get_app()
```

Comment:
```suggestion
    app = get_app_model()
```

Because of changes from #7269

Next argument for merge queue.